app/controllers/authenticate_controller.rb

```diff
@@ -3,6 +3,60 @@
 class AuthenticateController < ApplicationController
   include BasicAuthenticator
   include AuthorizeResource
+  include CurrentUser
+  include FindResource
+  include AssumedRole
+
+  def list_authenticators
```

Comment:
This method feels kind of weird here in that it's specific to OIDC, but the name doesn't give that indication.  As it's OIDC specific, what do you think about moving this into an OIDC specific controller?  

Historically we don't do this, but the `AuthenticateController` is a mess and handles a lot of authenticator specific behavior via methods.